bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/LaunchBarListener.java

```diff
@@ -49,6 +53,41 @@ public static void setIgnoreTargetChange(boolean status)
 		targetChangeIgnored = status;
 	}
 
+	@Override
+	public void activeLaunchDescriptorChanged(ILaunchDescriptor descriptor)
+	{
+		descriptor.getAdapter(ILaunchConfiguration.class);
```

Comment:
Do you want to remove this as we have not using the returned adapter?

Reply:
done